build-in-container/SPECS/hello_world_demo/hello_world_demo.spec

```diff
@@ -0,0 +1,35 @@
+Summary:        A demonstration package that creates a "Hello-World" application
```

Comment:
Wby do we need multiple copies of these?

Reply:
This is to show an example spec file. I can remove this example, and instead point the user to it in the README. Does that sound better?

Reply:
Lets prioritize making this easy for someone to checkout and try. Any simple spec will work to illustrate the structure.

Reply:
Going with Sriram's recommendation, and leaving the example spec here

Reply:
I'm still very confused why the version of this [here](https://github.com/microsoft/CBL-MarinerTutorials/blob/main/SPECS/hello_world_demo/hello_world_demo.spec) isn't sufficient for this. With the new version able to point at other directories I'd assume the existing version should be ok now?

Reply:
Yes, the tool now lets the user build SPECS from another Mariner directory. But, if it's a new user unfamiliar with the Mariner specs, I think this will be very useful as a sample. I don't see any harm here, but we can remove it in the next revision, if it proves useless

Reply:
Having duplicate copies of the exact same files in two spots is incredibly confusing to me. The end user will not have a set of packages checked into the tutorial repo, they will have their own repo with their own /SPECS folder. The directions should try and explain a flow that allows them to build those packages. Using the existing version of the hello world package is a really easy way to do this. (and also allows us to use the existing .gitignore paths since it will build in the same spot as all the other tutorials in the repo). This partly ties into the default path not being the current path which I fundamentally disagree with. A tool should be expected to operate on the folder the user is in, unless explicitly told otherwise.